features/rails_features/breadcrumbs.feature

```diff
@@ -56,3 +56,24 @@ Scenario Outline: SQL Breadcrumb
     | 2.4          | 5             |
     | 2.5          | 3             |
     | 2.5          | 5             |
+
+Scenario Outline: Mongo breadcrumbs
```

Comment:
I would put this in a separate feature file as it isn't related to the other activesupport/Rails related breadcrumbs.